lib/acts_as_list/active_record/acts/position_column_method_definer.rb

```diff
@@ -69,21 +69,14 @@ def self.define_instance_methods(caller_class, position_column)
       end
 
       define_method :touch_record_sql do
-        cached_quoted_now = quoted_current_time_from_proper_timezone
+        time = (default_timezone == :utc ? Time.now.utc : Time.now)
```

Comment:
Should this be `self.class.default_timezone`? per what Rails does? Also, I'd feel more comfortable deferring to their method: `current_time_from_proper_timezone` since then, if it ever changes in Rails we won't have to detect and change also.

Reply:
default_timezone is a class attribute accessor, like class variable, so it can be access by every AR instance.
agreed with you that `self.class.default_timezone` is more clear.